### DISCUSSION TITLE
Use the full screen width in 'auto' layout?

### DISCUSSION THREAD

**Discussion by attzonko** [Lissy93/dashy#554] [Help and Support] [answered]
First, thank you for this amazing tool!

Question: Is it possible to configure the 'auto' layout to use the full screen? On my ultrawide monitor, the auto mode for me ends up using only ~33% of my screen real estate. I am not sure if it is a user error with respect to configuration, but I would love to be able to use my full screen real estate.

**Comment by lissy93** [comment, marked as answer]
I'm glad your enjoying it :)

Yes, this is possible. I think this is something I will improve at some point as a [similar question](https://github.com/Lissy93/dashy/discussions/148#discussioncomment-1889797) has previously been asked. But in the meantime, you can add something like this to your custom CSS to `appConfig.customCss`.

```css
.item-group-container { max-width: 90%; }
```

**Reply by attzonko** [reply]
Thanks for the quick response. I tried this suggestion, but I think it gets overridden.  the `max-width: 90%` is already in the style sheets. However I think the `.item-group-container.orientation-auto.item-size-small` ends up setting the `max-width` to 1400px.

I hope I am using the correct terms, I am a Firmware programmer first, so the CSS stuff is still somewhat magic to me...

**Reply by attzonko** [reply]
![image](https://user-images.githubusercontent.com/4975715/158072784-76015de4-6c60-48ef-952a-5e0be9e547ae.png)

**Reply by lissy93** [reply]
Oops, yeah looks like you're correct, try:
`.item-group-container { max-width: 100% !important; }` - That should do it 😛

**Reply by attzonko** [reply]
That did it, and I learned something `!important` :P thanks!
